.github/workflows/build-and-push-rag-content.yaml

```diff
@@ -0,0 +1,106 @@
+name: "Build and push rag-content container image"
+
+on:
+  workflow_dispatch:
+
+  pull_request:
+    branches:
+      - main
+    types:
+      - opened
+      - edited
+      - synchronize
+      - reopened
+
+  schedule:
+    - cron: '15 02 * * *'
+
+env:
+  EXTRA_BUILD_ARGS: ""
+  IMAGE_NAME: quay.io/openstack-lightspeed/${{ github.event.repository.name }}
+
+jobs:
+  build-and-push-rag-content:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Free disk space (Ubuntu)
+        uses: jlumbroso/free-disk-space@v1.3.1
+        with:
+          # This might remove tools that are actually needed,
+          # if set to "true" but frees about 6 GB.
+          tool-cache: true
+
+      - name: Checkout to the repository
+        uses: actions/checkout@v4
+
+      - name: Install podman
+        run: |
+          sudo apt update
+          sudo apt install podman -y
+          sudo apt install jq -y
+
+      - name: Install cosign
+        uses: sigstore/cosign-installer@v3.8.1
+
+      - name: Log in to registry.redhat.io
+        uses: redhat-actions/podman-login@v1
+        with:
+          username: ${{ secrets.REDHAT_REGISTRY_LOGIN }}
+          password: ${{ secrets.REDHAT_REGISTRY_PASSWORD }}
+          registry: registry.redhat.io
+
+      - name: Build only partial rag-content when running in the check pipeline
+        if: github.event_name != 'schedule'
+        run: |
+          echo EXTRA_BUILD_ARGS=OS_PROJECTS="nova" >> $GITHUB_ENV
```

Comment:
You don't have to use the environment file directly, instead use workflow variable https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/store-information-in-variables

Reply:
Thanks, I'm going to add this.

Reply:
Hmmm, I tried this one but without success. Did you mean something like this?

```
- name: Build only partial rag-content when running in the check pipeline
  if: github.event_name != 'schedule'
  env: 
      EXTRA_BUILD_ARGS: OS_PROJECTS="nova"
```

The `EXTRA_BUILD_ARGS` var in this case is available only in the step in which it is defined.

Reply:
I removed the EXTRA_BUILD_ARGS as discussed earlier. I'm marking this as resolved.